src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-details/osd-details.component.html

```diff
@@ -17,9 +17,9 @@
     </cd-table-performance-counter>
   </tab>
   <tab heading="Histogram">
-    <h3 *ngIf="osd.loaded && osd.histogram_failed">
-      Histogram not available -> <span class="text-warning">{{ osd.histogram_failed }}</span>
-    </h3>
+    <h4 class="alert alert-warning" *ngIf="osd.loaded && osd.histogram_failed">
```

Comment:
Could you use `warning-panel` here? That way we would have all the warning with the same look and feel.